packages/flutter_tools/templates/cocoapods/Podfile-ios-objc

```diff
@@ -67,7 +67,7 @@ target 'Runner' do
   # referring to absolute paths on developers' machines.
   system('rm -rf .symlinks')
   system('mkdir -p .symlinks/plugins')
-  plugin_pods = parse_KV_file('../.flutter-plugins')
+  plugin_pods = parse_KV_file('./Flutter/.flutter-plugins')
```

Comment:
The Podfiles aren't owned by Flutter, and they get generated only when the app is created.  So existing Podfiles are going to expect '../.flutter-plugins' and won't get this change without a migration.

Reply:
Agreed. I am still generating the current file at the same location to avoid breaking existing apps. I'll add a warning message if they are using the "old" podfile so the user can delete it and auto-generate it again.

Reply:
Can you copy/make the symlinks in this Ruby file from the old one if the old directory is present and the new one isn't so they don't need to migrate anything, and a `pod install` won't just fail?  The `unless` block right above is doing a similar one-time `cp` migration

Reply:
Or something like this so they know what to do if the new file doesn't exist:
```ruby
plugin_file = File.join(__dir__, 'Flutter', '.flutter-plugins')
unless File.exist?(plugin_file)
  raise "Flutter/.flutter-plugins must exist. If you're running pod install manually, make sure flutter pub get is executed first"
end
plugin_pods = parse_KV_file(plugin_file)
```